hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/state/ProxyWorldUpdaterTest.java

```diff
@@ -253,6 +253,7 @@ void providesAccessToPendingStorageChanges() {
     void cannotCreateUnlessPendingCreationHasExpectedAddress() {
         given(hederaOperations.peekNextEntityNumber()).willReturn(NEXT_NUMBER);
         given(hederaOperations.contractCreationLimit()).willReturn(1234L);
+        given(hederaOperations.accountCreationLimit()).willReturn(1234L);
```

Comment:
Gosh, maybe `4321L` just to distinguish it from the contract creation limit?  Probably doesn't matter.